README.md

```diff
@@ -31,6 +31,27 @@ end
 
 NOTE: If you don't already have an environment variable for your Mandrill API key, don't forget to create one.
 
+### Mandrill Templates
+
+If you want to use this gem with mandrill templates you just have to add the `template` param to the `mail` function.
+
+```ruby
+class MyMailer < ActionMailer::Base
+  def notify_user(email)
+    mail(
+      to: email,
+      from: 'your@email.com',
+      template: 'your-mandrill-template-slug',
+      template_content: { # optional
```

Comment:
@kshnurov the `mandrill_api` gem also accepts this syntax, if you prefer i can change it but i think its more clear this way. What do you think?

https://bitbucket.org/mailchimp/mandrill-api-ruby/src/044396ea8f760a617a1b62e28f69b77833d82d55/lib/mandrill/api.rb?at=master&fileviewer=file-view-default#api.rb-848